jetty-ee10/jetty-ee10-maven-plugin/src/it/it-parent-pom/pom.xml

```diff
@@ -17,7 +17,7 @@
       <dependency>
         <groupId>commons-io</groupId>
         <artifactId>commons-io</artifactId>
-        <version>2.7</version>
+        <version>2.14.0</version>
```

Comment:
If I remember correctly we should be able to use the property from top pom:
 `${commons-io.version}`  or `@commons-io.version@`